testkit-backend/backend.go

```diff
@@ -1471,6 +1480,45 @@ func serializeNotifications(slice []neo4j.Notification) []map[string]any {
 	return res
 }
 
+func serializeGqlStatusObjects(slice []neo4j.GqlStatusObject) []map[string]any {
+	if slice == nil {
+		return []map[string]any{}
+	}
+	if len(slice) == 0 {
+		return []map[string]any{}
+	}
```

Comment:
The first is a special case of the second. So only the second would be required, if I read this right.

Further, doesn't `range` over a `nil` map simply execute 0 times making the whole function just work for `nil` without any guards at the top?

Reply:
Yep think I added the guard before I refactored the function to return an empty map by default so will work fine without the guards. Updated